atomic_reactor/schemas/container.json

```diff
@@ -82,9 +82,42 @@
         "inherit": {
           "description": "enable inheritance of yum repourls and composes from baseimage build (disabled by default)",
           "type": "boolean"
+        },
+        "no_deps": {
+          "description": "do not resolve dependencies and only use the listed modules or packages",
+          "type": "boolean"
+        },
+        "check_deps": {
+          "description": "abort the compose if any package or module has a broken dependency",
+          "type": "boolean"
+        },
+        "include_unpublished_pulp_repos": {
+          "description": "include unpublished repos in pulp input content-sets",
+          "type": "boolean"
+        },
+        "multilib_method": {
+          "description": "list of options to decide if a package is multilib",
+          "type": ["array", "null"],
+          "items": {
+              "enum": ["none", "iso", "runtime", "devel", "all"]
+          }
+        },
+        "multilib_arches": {
+          "description": "tags of modular Koji content to include in the build",
+          "type": ["array", "null"],
+          "items": {
+              "type": "string"
+          }
+        },
```

Comment:
Given that we don't do anything with these properties, do we really want to allow them in container.yaml?

Reply:
we support multlib_arches and multilib_methods.  See #1084.

Fixed the description, though.

Reply:
Ah, thanks. and I see include_unpublished_pulp_repos is also used.

What about no_deps and check_deps?

Reply:
They're options for the flags argument same as include_unpublished_pulp_repos and use the same syntax.

Reply:
but I don't see those 2 options used anywhere

Reply:
pre_resolve_composes scoops up everything that isn't used somewhere else into flags and passed them to ODCS as a flag.

Reply:
this is only place where any flags are set

https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/pre_resolve_composes.py#L378

Reply:
What's the resolution to this?

Reply:
They're valid flags, I'd like to keep them in the schema even if we don't support them in pre_resolve_composes.

Reply:
Removed the flags we don't support.